activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+*   `alias_attribute` can be used to alias columns named with a primary key reserved terms.
```

Comment:
It's not just primary keys though right? We're improving alias attribute for any dangerous attribute method so I think we should mention that.

Reply:
Technically yes, we made it possible to use `alias_attribute` with more than just pk-related terms, for example:

```ruby
alias_attribute :get_update_value, :update
alias_attribute :get_save_value, :save
alias_attribute :get_destroy_value, :destroy
```

But it's tricky because Rails won't let you to boot an app with columns named `update`, `destroy` or `update`. 
So it will be possible to use this feature without changes to the `alias_attribute` logic if Rails loosens the following restriction:
https://github.com/rails/rails/blob/2e181b1b2edbeed0c6afb11cc037dd23c985fdb9/activerecord/lib/active_record/attribute_methods.rb#L89

I can change the description to cover all reserved methods as it is correct from technical standpoint but it won't be fair as there will be no applications with columns named with most of reserved terms